pkg/reconciler/common/prune.go

```diff
@@ -345,19 +347,24 @@ func createCronJob(ctx context.Context, kc kubernetes.Interface, cronName, targe
 }
 
 func pruneCommand(pru *pruneConfigPerNS, ns string) string {
```

Comment:
1. rename this function to something like generatePruneConfig as it is not creating prune command instead it is just accumulating the configuration for each namespace.

2. this function needs a detailed comment(s) either at the start of the function or within. In comments mention a sample configuration string representing atleast 3 namespaces (output of this function).

I believe these changes are important to make it evident what this patch is improving.

Reply:
:+1: